LoopKitUI/Views/Settings Editors/CorrectionRangeOverrideReview.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CorrectionRangeOverrideReviewView.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/1/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+import HealthKit
+
+public struct CorrectionRangeOverrideReview: View {
+    @ObservedObject var viewModel: TherapySettingsViewModel
+    private let mode: PresentationMode
+    private var unit: HKUnit {
+        self.viewModel.therapySettings.glucoseUnit!
+    }
+
+    public init(mode: PresentationMode = .acceptanceFlow, viewModel: TherapySettingsViewModel) {
+        precondition(viewModel.therapySettings.glucoseUnit != nil)
+        precondition(viewModel.therapySettings.glucoseTargetRangeSchedule != nil)
+        self.mode = mode
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        CorrectionRangeOverridesEditor(
+            value: CorrectionRangeOverrides(
+                preMeal: viewModel.therapySettings.preMealTargetRange,
+                workout: viewModel.therapySettings.workoutTargetRange,
+                unit: unit
+            ),
+            unit: unit,
+            correctionRangeScheduleRange: (viewModel.therapySettings.glucoseTargetRangeSchedule?.scheduleRange())!,
```

Comment:
[nit] You can drop the parens and move the force unwrap onto `glucoseTargetRangeSchedule` since that's where the optionality is

Reply:
(code was copied from elsewhere...)